trace/opencensus/trace/exporters/file_exporter.py

```diff
@@ -27,14 +28,19 @@ class FileExporter(base.Exporter):
     :param file_name: The name of the output file.
     """
 
-    def __init__(self, file_name=DEFAULT_FILENAME):
+    def __init__(self, file_name=DEFAULT_FILENAME,
+                 transport=sync.SyncTransport):
         self.file_name = file_name
+        self.transport = transport(self)
 
-    def export(self, trace):
+    def emit(self, trace):
```

Comment:
Can you add emit() to base.Exporter, and add doc comments explaining emit() and export()?

The naming is a little weird, since I don't usually think of "transport" for something that's just printing or appending to a file.  But I don't have a better suggestion.

Reply:
Done.